src/style.css

```diff
@@ -1,35 +1,34 @@
 @tailwind base;
 
 body {
-  @apply bg-dark-blue p-12 text-mid-grey;
+  @apply bg-hack-dark-bg p-12 text-hack-fg;
   font-family: 'Space Mono', monospace;
   padding: 0;
 }
 
 @screen light-mode {
   body {
-    @apply bg-white;
+    @apply bg-hack-fg;
```

Comment:
What about using `#f4f0e1` which is the same colour as the background on https://hacktoberfest.digitalocean.com/? It would prob mean a new variable `bg-hack-light` or something. In fact shouldn't this be `bg-hack-alt-bg`?? If so, should we not make that `#f4f0e1` ?

Reply:
Ah I see `hack-alt-bg` is more the secondary / contrasting colour. So maybe a new var `hack-bg-light` or something?

Reply:
For Sure, I created a new variable called '**hack-light-bg**' and applied him to the body of light-mode;